aapp_runner/read_aapp_config.py

```diff
@@ -1,12 +1,60 @@
-import os
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021 Adam.Dybbroe
+
+# Author(s):
+
+#   Adam.Dybbroe <a000680@c21856.ad.smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
 
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Reading and checking yaml file configurations
+"""
+
+import os
 from socket import gethostname, gethostbyaddr, gaierror
 
+
+class StationError(Exception):
+    pass
+
+
+class EnvironmentError(Exception):
+    pass
+
+
+class AappProcessKeyMissing(Exception):
+    pass
+
+
+class AappWorkDirNotSet(Exception):
+    pass
+
+
+class ConfigFileOptionsError(Exception):
+    pass
+
+
+class StaticConfigError(Exception):
+    pass
```

Comment:
Should these be inheriting from RuntimeError instead? Also docstrings should be added.

Reply:
Good point!